Libraries/test/TestServerlessApp.IntegrationTests/IntegrationTestContextFixture.cs

```diff
@@ -14,24 +14,24 @@
 
 namespace TestServerlessApp.IntegrationTests
 {
-    public class IntegrationTestContextFixture : IDisposable
+    public class IntegrationTestContextFixture : IAsyncLifetime
     {
-        private readonly string _stackName;
-        private readonly string _bucketName;
-        private readonly CloudFormationHelper _cloudFormationHelper;
-        private readonly S3Helper _s3Helper;
+        private string _stackName;
+        private string _bucketName;
+        private CloudFormationHelper _cloudFormationHelper;
+        private S3Helper _s3Helper;
 
-        public readonly LambdaHelper LambdaHelper;
-        public readonly CloudWatchHelper CloudWatchHelper;
-        public readonly string RestApiUrlPrefix;
-        public readonly string HttpApiUrlPrefix;
-        public readonly List<LambdaFunction> LambdaFunctions;
-        public readonly HttpClient HttpClient;
+        public LambdaHelper LambdaHelper;
+        public CloudWatchHelper CloudWatchHelper;
+        public string RestApiUrlPrefix;
+        public string HttpApiUrlPrefix;
+        public List<LambdaFunction> LambdaFunctions;
+        public HttpClient HttpClient;
```

Comment:
properties/fields like helpers which can be initialized in constructor should stay in constructor.

Move only the network call/async operations to `InitializeAsync`

Reply:
Fixed